lib/r2d2/payment_token.rb

```diff
@@ -57,7 +57,6 @@ def decrypt_message(encrypted_data, symmetric_key)
         decipher = OpenSSL::Cipher::AES128.new(:CTR)
         decipher.decrypt
         decipher.key = symmetric_key
-        decipher.auth_data = ""
```

Comment:
This change will, I believe, break forward Ruby compatibility?

Reply:
The current tests in place all pass with this line removed. I must admit I didn't look any further than that. If it does break forward ruby compatibility we should be able to put together a test that confirms this.

Reply:
This is safe to remove, AES-CTR is not an AEAD mode so it didn't make sense anyway. Older OpenSSL gems just didn't complain about it. Starting with OpenSSL gem 2.0 it started getting stricter (e.g. raising error on too long keys instead of truncating).